src/pages/MainPage/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React, {useEffect} from 'react'
+import { useAppDispatch, useAppSelector } from '../../hooks/hooks'
+import { fetchFilms } from '../../store/slices/filmSlice'
+import { ContentWrapper } from '../../components/ContentWrapper'
+import { MainCardItem } from '../../components/MainCardItem'
+import { IFilms } from '../../types/IFilms'
+import './styles.scss'
+
+const MainPage: React.FC = () => {
```

Comment:
```suggestion
const MainPage = () => {
```
не используй FC для обозначения компонентов, есть сайд эффекты. Можно без него обойтись